docs/antrea-network-policy.md

```diff
@@ -365,19 +399,30 @@ Usage of ClusterGroups along with stand-alone selectors is not allowed.
 
 ### Behavior of *to* and *from* selectors
 
-There are four kinds of selectors that can be specified in an ingress `from`
+There are five kinds of selectors that can be specified in an ingress `from`
 section or egress `to` section:
```

Comment:
I wonder why we didn't add ClusterGroup to this list?

Reply:
oversight? laziness? rebel?